frontend/src/pages/members/MemberDependantsTab.tsx

```
@@ -0,0 +1,189 @@
+              style={{ 
+                backgroundColor: colors.background.secondary, 
+                borderColor: colors.border.primary 
+              }}
+              onMouseEnter={(e) => {
+                e.currentTarget.style.backgroundColor = colors.background.tertiary;
+              }}
+              onMouseLeave={(e) => {
+                e.currentTarget.style.backgroundColor = colors.background.secondary;
+              }}
+              onClick={() => handleDependantClick(dependant.id)}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)